### PR TITLE
Astro 1319 static assets

### PR DESCRIPTION
**Pull request by Cheshire89**
https://rocketcom.atlassian.net/browse/ASTRO-1319

As a developer I want to *be able to download and install compiled versions of Astro’s static assets so that I can use them in projects that don't make use of Astro's web components.

1. Update to use scss
2. create build process for rux-core module which includes compiled static assets css & scss
3. find a way to inject scss into components to remove css code duplication
4. Update storybook to 6+

++ fix any of the associated bugs with each of the steps

### CODE REVIEW

File: src/components/rux-sign-in/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   "module": "rux-sign-in.js",
   "name": "@astrouxds/rux-sign-in",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo testing rux-sign-in, version: $npm_package_version"
```
Comment (Full-lifey) [inline src/components/rux-sign-in/package.json:27]: I like the addition of a test

File: src/scss/common/_fonts.scss [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+
+@font-face {
+  font-family: "Roboto Mono";
+  src: url("../fonts/roboto-mono-regular.woff2") format("woff2"),
+    url("../fonts/roboto-mono-regular.woff") format("woff");
+  font-weight: 400;
+  font-style: normal;
+}
```
Comment (Full-lifey) [inline src/scss/common/_fonts.scss:114]: 🙏  🥳

**Review by Full-lifey — APPROVED** [review]
I noticed one tiny thing but otherwise looks great!

File: src/components/rux-sign-in/package.json (hunk 1 above)
Comment (Cheshire89) [reply src/components/rux-sign-in/package.json:27]: this was for lerna config

**Review by cskerbo — APPROVED** [review]
Excited to finally check this one off the list! Good job, I did not notice any glaring issues. 

🥳

File: .storybook/main.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+const path = require('path');
```
[the remark this answers is not in the document]
Comment (Cheshire89) [reply .storybook/main.js:1]: not anymore ... will remove

**Review by dmcalester — APPROVED** [review]
Looks good Sasha. Thank you

**Pull request merged**